src/Suggestions/Suggestions.js

```diff
@@ -0,0 +1,107 @@
+import styles from './Suggestions.module.css';
+import React from 'react';
+import { Avatar } from '../Avatar/Avatar';
+import { NavBar } from '../NavBar/NavBar';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+
+export function Suggestions({ isExpanded }) {
+  const [suggestionsData, setSuggestionsData] = React.useState([]);
+  React.useEffect(() => {
+    fetch('/Data/suggestions.json', {
+      method: 'GET',
+    })
+      .then((res) => res.json())
+      .then((results) => setSuggestionsData(results));
+  });
+  const slicedSuggestionData = suggestionsData.slice(0, 5);
+
+  return (
+    <>
+      <NavBar />
```

Comment:
You're showing the navbar even inside the component when it's not expanded. You'll have two navbars this way.

That's why I suggested creating a `SuggestionsPage`, so it will have:

```jsx
<NavBar />
<Suggestions isExpanded />
```

Reply:
Done